#! /bin/sh /usr/share/dpatch/dpatch-run
## 16_loose_spelling_fix.dpatch by William Grant <william.grant@ubuntu.org.au>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix two instances of 'loose' to 'lose'. (LP: #61952)

@DPATCH@
diff -urNad terminatorx-3.82~/src/tX_mastergui.cc terminatorx-3.82/src/tX_mastergui.cc
--- terminatorx-3.82~/src/tX_mastergui.cc	2004-11-04 00:25:50.000000000 +1100
+++ terminatorx-3.82/src/tX_mastergui.cc	2007-04-01 19:13:55.000000000 +1000
@@ -265,7 +265,7 @@
 GtkSignalFunc new_tables() {
 	GtkWidget *dialog=gtk_message_dialog_new(GTK_WINDOW(main_window), 
 		GTK_DIALOG_DESTROY_WITH_PARENT, GTK_MESSAGE_QUESTION, GTK_BUTTONS_YES_NO,
-		"Are you sure you want to loose all turntables and events?");
+		"Are you sure you want to lose all turntables and events?");
 	
 	int res=gtk_dialog_run(GTK_DIALOG(dialog));
 	gtk_widget_destroy(dialog);
@@ -883,7 +883,7 @@
 	if (globals.quit_confirm) {
 		GtkWidget *dialog=gtk_message_dialog_new(GTK_WINDOW(main_window), 
 		GTK_DIALOG_DESTROY_WITH_PARENT, GTK_MESSAGE_QUESTION, GTK_BUTTONS_YES_NO,
-		"Exit terminatorX and loose all unsaved data?");
+		"Exit terminatorX and lose all unsaved data?");
 		
 		int res=gtk_dialog_run(GTK_DIALOG(dialog));
 		gtk_widget_destroy(dialog);
